process/transaction/shardProcess.go

```diff
@@ -612,6 +614,35 @@ func (txProc *txProcessor) addFeeAndValueToDest(acntDst state.UserAccountHandler
 	return txProc.accounts.SaveAccount(acntDst)
 }
 
+func (txProc *txProcessor) processRelayedTxV3(
+	tx *transaction.Transaction,
+	relayerAcnt, acntDst state.UserAccountHandler,
+) (vmcommon.ReturnCode, error) {
+	if !txProc.enableEpochsHandler.IsRelayedTransactionsV3FlagEnabled() {
+		return vmcommon.UserError, txProc.executingFailedTransaction(tx, relayerAcnt, process.ErrRelayedTxV3Disabled)
```

Comment:
this has to be backward compatible. So let's make sure that TX is not accepted in the pool if it is of RelayedV3 and the flag is not active. This check should be done at transaction interceptor level.

The problem is nodes with different software will run at the same time in the first few days of the announcement of the new release.

Reply:
added